third_party/webports/README.google

```diff
@@ -5,3 +5,6 @@ License File: LICENSE
 Description:
 Collection of open source libraries and applications that have been ported to
 Native Client, along with set to tools for building and maintaining them.
+Local Modifications:
+Changed the PIP installation script to work around the import error.
```

Comment:
nit: Either change to "an import error" or quickly explain the error / link to the issue.

Reply:
done